docs/configuration.md

```diff
@@ -55,7 +55,7 @@ Generated keypair with encoded pubkey: YOUR_PUBLIC_KEY
 Generated public-private key pair will be located in `/tmp/node/generated_keys`, and public key is to be put in epoch configuration file (`mining` > `beneficiary` parameter).
 
 Do make sure you back up `/tmp/node/generated_keys` (and remember the password): if you destroy the node, you can setup a new node and use the same account (public key) as a beneficiary.
-You shall not share the private key (or the password) with anyone.
+**You shall not share the private key (or the password) with anyone.**
```

Comment:
I think we should rather stress and highlight/bold
```
NOTE: This way of generating a key-pair is only for testing purpose, use a proper wallet/mechanism for your mainnet tokens!
```
in https://github.com/aeternity/epoch/blob/4619527562c168e9a6efa59ccbfd6fc26ed9c990/docs/configuration.md#beneficiary-account section.